bears/general/LineLengthBear.py

```diff
@@ -30,7 +46,13 @@ def run(self,
         :param indent_size:         Number of spaces per indentation level.
         :param ignore_length_regex: Lines matching each of the regular
                                     expressions in this list will be ignored.
+        :param language:            Programming language of the source code.
         '''
+        if 'max_line_length' in language.attributes:
+            max_line_length = language.max_line_length
+        else:
+            language = Language['Unknown']
```

Comment:
If the user has specified a language, we should respect that when we create the `Result`.